!GameTexts/NewTexts.xml

```diff
@@ -608,19 +605,19 @@
 			<Text>Khopesh</Text>
 		</Row>
 		<Row Tag="TXT_KEY_UNIT_EGYPTIAN_KHOPESH_TEXT">
-			<Text>Khopesh is the Egyptian name of the Canaanite "sickle-sword", in Assyrian known as sappara. Its origins can be traced back to Sumer of the third millennium BC.[NEWLINE][NEWLINE]A typical khopesh is 50–60 cm (20–24 inches) in length, though smaller examples do also exist. This blade was designed for hooking an opponent's shield or disarming them. These weapons changed from bronze to iron in the late period.[NEWLINE][NEWLINE]The blade is only sharpened on the outside portion of the curved end. The khopesh evolved from the epsilon or similar crescent shaped axes that were used in warfare. Note, however, that the khopesh is not an axe. Unlike an axe, the khopesh did not make push-cuts, but rather slashes, like a sabre. The khopesh went out of use around 1300 BC.</Text>
+			<Text>Khopesh is the Egyptian name of the Canaanite "sickle-sword", in Assyrian known as sappara. Its origins can be traced back to Sumer of the third millennium BC. A typical khopesh is 50–60 cm (20–24 inches) in length, though smaller examples do also exist. This blade was designed for hooking an opponent's shield or disarming them. These weapons changed from bronze to iron in the late period.[NEWLINE][NEWLINE]The blade is only sharpened on the outside portion of the curved end. The khopesh evolved from the epsilon or similar crescent shaped axes that were used in warfare. Note, however, that the khopesh is not an axe. Unlike an axe, the khopesh did not make push-cuts, but rather slashes, like a sabre. The khopesh went out of use around 1300 BC.</Text>
 		</Row>
 		<Row Tag="TXT_KEY_UNIT_EGYPTIAN_KHOPESH_STRATEGY">
 			<Text>The Khopesh is the unique Egyptian replacement for the Spearman. It's a strong early unit that has an edge in direct combat. However, unlike the Spearman it replaces, it does not have a combat bonus against mounted units.</Text>
 		</Row>
 		<Row Tag="TXT_KEY_UNIT_EGYPTIAN_KHOPESH_HELP">
-			<Text>Strong, front-line land Unit of the Ancient Era. Only Egypt may build it. It has the Fatigue promotion, which temporarily reduces the combat power of a unit that it attacks.</Text>
+			<Text>Strong, front-line land Unit of the Ancient Era. Only Egypt may build it. Deals increased damage vs heavily damaged units, and has the Skillful promotion, which temporarily reduces the combat power of a unit that it attacks.</Text>
 		</Row>
 		<Row Tag="TXT_KEY_PROMOTION_UNIT_EGYPT_FATIGUE">
-			<Text>Skillful Warrior</Text>
+			<Text>Skillful</Text>
 		</Row>
 		<Row Tag="TXT_KEY_PROMOTION_UNIT_EGYPT_FATIGUE_HELP">
-			<Text>[COLOR_POSITIVE_TEXT]After Attack[ENDCOLOR] Defending Enemy Unit gets -20% [ICON_STRENGTH] Combat Strength [COLOR_POSITIVE_TEXT]until it Fortifies[ENDCOLOR].</Text>
+			<Text>[COLOR_POSITIVE_TEXT]After Attack[ENDCOLOR] Defending Enemy Unit gets -20% [ICON_STRENGTH] [COLOR_POSITIVE_TEXT]until it Fortifies[ENDCOLOR].</Text>
```

Comment:
"Combat Strength" is key word I kepp using to have some pattern. Bring it back please.

Reply:
Current VP promotions do not write out combat strength in many cases in order to save space.  I know you are planning to overhaul that, so do whatever is consistent with your edits